tools/dependency/release_dates.py

```diff
@@ -132,19 +183,27 @@ def verify_and_print_release_dates(repository_locations, github_instance):
 
 
 if __name__ == '__main__':
-    if len(sys.argv) != 2:
+    # github action will have 3 args and CI will have 2 args
+    if len(sys.argv) < 1 or len(sys.argv) > 3:
         print('Usage: %s <path to repository_locations.bzl>' % sys.argv[0])
         sys.exit(1)
+    # parsing location and github_action flag with argparse
+    parser = argparse.ArgumentParser()
+    parser.add_argument('location', type=str)
+    parser.add_argument('--cron', action='store_true')
```

Comment:
I find the name of this flag confusing, and not overly helpful

could we change it to something like `--create-issues`

Reply:
also, can we make this consistent throughout, currently it changes from `cron` -> `cron_action` -> `create_issue` 

and this is then used by a function called `create_issues`

Reply:
we have  a `create_issues` func already .I think have to change `--cron` flag to something else and also make it consistent too.